test/fw/ptl/lib/ptl_wrappers.py

```diff
@@ -2764,16 +2764,35 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
         if id is None and obj_type == SERVER:
             id = self.pbs_conf['PBS_SERVER']
         bs_list = []
-        if cmd == MGR_CMD_DELETE and oid is not None and rc == 0:
+        deleted = False
+        if cmd == MGR_CMD_DELETE and oid is not None:
```

Comment:
This cmd and elif command (line 2788) can be merged:
```
if cmd == MGR_CMD_DELETE and oid is not None:
    if obj_type == MGR_OBJ_RSC:  # check for obj_type here
        # new code here
    if rc == 0:
        # older code here
```

You can also get rid of the `deleted` variable

Reply:
this is new condition,  condition line 2788 is not used for resources. now we are not checking rc==0 for resources deletion.